lib/js/stanford_events.behavior.js

```diff
@@ -21,6 +21,40 @@ export default {
           $(this).attr('aria-expanded', 'false');
         }
       });
+
+      // Get today's date
+      var today = new Date();
+
+      // Update so today is all of the day
+      var a = today.setHours(0, 0, 0, 0);
+      var todayBegin = parseInt(a, 10);
+
+      // Location for label
+      const eventType =
+        $('div.node-stanford-event-su-event-type .su-event-type');
+
+      // Location for text
+      const eventTextLocation =
+        $('div.node-stanford-event-su-event-date-time');
+
+      // Looking for the date on the node page
+      const eventDate =
+        $('section.event').attr('data-end-date');
+
+      // Build the labels and sentence
+      const pastLabel =
+        $('<span class="su-event-label-past">Past Event ' +
+          '<span class="divider">|</span></span>');
```

Comment:
How does a screen reader read a pipe?

Reply:
Some say 'vertical line'. Here is a blog post  linked from WebAIM so I think it credible. https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/